### ISSUE TITLE
Find a better prop to represent the empty 3d space on new part (named views)

### ISSUE THREAD

**Issue by pierremtb** [KittyCAD/modeling-app#4217]
Seen it multiple time on v0.26.0 today

![Image](https://github.com/user-attachments/assets/ea2be722-8d48-4eb2-9491-a292a1809fb6)

The current 'feature' of showing the three planes on new part without entering sketch mode can be very confusing, especially with zoom

**Comment by pierremtb** [comment]
Updated description and title as discussed live @franknoirot

**Comment by franknoirot** [comment]
A mockup of some more specially-styled default planes can be seen [here in Figma](https://www.figma.com/design/5UUPOawPLEs3D0Gptf42vy/Designs?node-id=2965-12498&node-type=frame&t=KNvbFzJv2xhzJg0W-11), with an exported image below. For the teammate that picks this up, this will require a bit of checking with engine for how to get the text annotations (I believe we have the ability to do at least some light version of the mockup). Here are the requirements:
1. The planes remain the same (small) scale no matter how the user zooms. We need to do away with the "scale jump" that occurs with these planes when zooming in or out
2. The planes should have some text annotating them in opposite corners, one with the colloquial name and the other with the mathematical one. We do not need to annotate the back faces, those are implied by seeing reversed text annotations.

![Image](https://github.com/user-attachments/assets/26881a7c-8513-4b64-be34-6295dc8c3f3b)

**Comment by pierremtb** [comment]
Blocked by https://github.com/KittyCAD/engine/pull/3017 and https://github.com/KittyCAD/engine/pull/3009

**Comment by franknoirot** [comment]
I think I should split out another issue for the actual work for not using engine-side planes for the orientation prop, what do y'all think? To my eyes this has become decidedly about fixing the "disorientation problem" with empty scenes just by having a better vantage. But I see other reasons for not using engine-side planes as the orientation props:
1. engine planes jump in scale when zooming, causing a different form of disorientation. We would like to have screen-space scale for our orientation prop
2. engine planes have no way to apply text labels, which we would like to have on our orientation prop.

These lead me to think we still need something like a THREE.js set of planes to for orientation and clicking purposes.

**Comment by franknoirot** [comment]
You're absolutely right @pierremtb. We should make the gizmo more helpful for identifying the planes, and open another engine issue for having the option to create planes that are screen space scaled.

**Comment by pierremtb** [comment]
Tracking only the parent issue for the v1 Project and Milestone. Clearing these fields here.

**Comment by pierremtb** [comment]
Alright we can close this now that #6604 is in!
